core/controller/src/main/scala/whisk/core/entitlement/Entitlement.scala

```diff
@@ -205,6 +205,10 @@ protected[core] abstract class EntitlementProvider(config: WhiskConfig, loadBala
                 info(this, s"not authorized: $r")
             case Failure(t) =>
                 error(this, s"failed while checking entitlement: ${t.getMessage}")
+        } flatMap {
+            if (_) {
```

Comment:
minor: want to provide a variable name like `isAuthenticated`? This reads a bit odd.